editor/src/components/canvas/ui-jsx.test-utils.tsx

```diff
@@ -4,12 +4,15 @@ try {
     loadStoredState: () => Promise.resolve(null),
     saveStoredState: () => Promise.resolve(),
   }))
+  jest.mock('../code-editor/console-and-errors-pane', () => ({
+    ConsoleAndErrorsPane: () => null,
```

Comment:
I am mocking out the `ConsoleAndErrorsPane` from all end-to-end tests because it was throwing _100,000 error messages_